repl/scala-2.10/src/main/scala/org/apache/spark/repl/SparkILoop.scala

```diff
@@ -132,6 +132,7 @@ class SparkILoop(
   @DeveloperApi
   var sparkContext: SparkContext = _
   var sqlContext: SQLContext = _
+  var sqlContextWithHive: Boolean = _
```

Comment:
I don't think we need an extra var for this, and we can get the config directly since nowhere else will use it.